ES/ESmy028.xml

```diff
@@ -594,7 +594,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           </incipit>
                         </msItem>
 
```

Comment:
in this ms ESmy028 the collation has issues
`Quire with id:q13 and n 12 is made of 9 leaves, but there is no futher information about the reason this number is odd`
(add info on s.l. after line 731)

Reply:
line 627 "some leave" -> "some leaves"

Reply:
ok